sdk/communication/communication-chat/test/public/utils/recordedClient.ts

```diff
@@ -44,7 +44,7 @@ export const recorderOptions: RecorderStartOptions = {
 export async function createTestUser(recorder: Recorder): Promise<CommunicationUserToken> {
   const identityClient = new CommunicationIdentityClient(
     assertEnvironmentVariable("COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING"),
-    recorder.configureClientOptionsCoreV1({})
+    recorder.configureClientOptions({})
```

Comment:
Looks like the tests are failing in playback because of the api-version mismatch.
- Either re-record the test so that the api-version matches 
- or update the test to use a recorder matcher, `recorder.setMatcher(<the-matcher-you-want>)` so that the test won't fail due to api-version mismatch